src/Components/Timer.tsx

```diff
@@ -34,28 +13,38 @@ const Timer = () => {
   const gamePaused = useGamePausedDocState();
   const weaponsStolen = useWeaponsStolenDocState();
   const time = new Date();
-  const [soundOn, setSoundOn] = useState<boolean>(true)
+  const { gameAudio, loadAndPlayEscapeSoundtrack, playWarningSound, musicOn, soundOn } = useAudio();
 
   useEffect(() => {
+
     if (weaponsStolen.length === 4) {
       gameAudio.pause();
-      playWarning()
 
-      const warningTimer = setTimeout(async () => {
-        gameAudio = loadEscapeSoundtrack()
-        gameAudio.play();
-      }, 5100);
+      if (!musicOn) { 
```

Comment:
i think you can rewrite lines 20-33

```
if (weaponsStolen.length === 4) {
   gameAudio.pause();
   loadEscapeSoundtrack();

   if (musicOn) {
      playWarningSound();
      const warningTimer = setTimeout(async () => {
         playSoundtrack();
      }, 5100);
      return () => clearTimeout(warningTimer);
}
```

ill suggest changes on the useSounds.tsx to cater for playSoundtrack()